pkg/controller/directvolumemigration/stunnel.go

```diff
@@ -546,24 +552,28 @@ func (t *Task) createStunnelClientPods() error {
 				Containers: containers,
 			},
 		}
-		t.Log.Info(fmt.Sprintf("Creating Stunnel Client Service [%v/%v] on source cluster",
-			svc.Namespace, svc.Name))
+		t.Log.Info("Creating Stunnel client Service on source cluster",
+			"service", path.Join(svc.Namespace, svc.Name))
 		err := srcClient.Create(context.TODO(), &svc)
 		if k8serror.IsAlreadyExists(err) {
-			t.Log.Info("Stunnel client svc already exists on source", "namespace", svc.Namespace)
+			t.Log.Info("Stunnel client Service already exists on source cluster",
+				"service", path.Join(svc.Namespace, svc.Name))
 		} else if err != nil {
 			return err
 		}
-		t.Log.Info("stunnel client svc created", "name", clientPod.Name, "namespace", svc.Namespace)
-		t.Log.Info(fmt.Sprintf("Creating Stunnel Client Pod [%v/%v] on source cluster",
-			svc.Namespace, svc.Name))
+		t.Log.Info("Stunnel client Service created",
```

Comment:
wondering whether we need both "creating" and "created" logs here

Reply:
yeah this is one of those places where the "created" log already existed and I just added the pre-create message. I'll take it out.

Reply:
Resolved